Automattic/woocommerce-payments
# Stop refund process when using an invalid amount

Fixes #2287 

#### Changes proposed in this Pull Request

Don't attempt to refund an order with an invalid amount

When entering an amount for a refund, or when clicking without entering anything, we were making the request to the server and returning an error, so we should catch this before making that request.

https://user-images.githubusercontent.com/2612426/127121785-f0e14b7d-c61e-4dd6-aed9-78a35fdee1d7.mp4

#### Testing instructions

* Go to an order
* Click on Refund
* Click on "Refund 0 using WooCommerce Payments"
* See error message without making API request

-------------------
